pokemongo_bot/__init__.py

```diff
@@ -341,18 +341,19 @@ def _set_starting_position(self):
                 if not self.config.location:
                     sys.exit("No cached Location. Please specify initial location.")
                 else:
-                    pass
+                    self._read_config_location()
```

Comment:
thats how it was before, however then the cached location will get overridden with whatever is in location